### ISSUE TITLE
Feature request - web login.

### ISSUE THREAD

**Issue by lyndon160** [furlongm/openvpn-monitor#38]
There are so many features that could be implemented for this project, as you know.

However, I think that it is especially important that there is some kind of authorisation for the web interface, so that at least to a primitive extent, access to it could be controlled.

**Comment by jomzki03** [comment]
up for this feature.

**Comment by tony-caffe** [comment]
Why would you not send this web interface over https using htpasswd? Seems like an easy way to mitigate in the mean time and still very secure. Allows you to control users via u/p and gives it encryption. 

```
<Directory "/example/path/">
...
   AuthName "Site OpenVPN Management Access"
   AuthType Basic
   AuthUserFile /etc/example/location/htpasswd.users
   Require valid-user
...
</Directory>
```

**Comment by TheAmigo** [comment]
I agree with @tony-caffe.  If you don't want to manage users vi .htpasswd, use AuthLDAP to point it at AD and require that the user be a member of a specific AD group to access the page.
